shell/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -96,6 +94,16 @@ ui::MouseWheelEvent UiMouseWheelEventFromWebMouseEvent(
           base::ClampFloor<int>(event.delta_y)};
 }
 
+// TODO(reito): Remove this function and use default 1.0f when breaking change
+// is announced for a sufficient period.
```

Comment:
_(nit)_ I don't think I have enough context from this PR to know what a sufficient period is. I think this is probably also gonna be true for maintainers who come across this comment in the codebase.

Please let's use "This can be removed in Electron ${version}" or "This can be removed after ${date}".